src/basic/missing_syscall.h

```diff
@@ -334,7 +334,7 @@ static inline pid_t raw_getpid(void) {
 #  elif _MIPS_SIM == _MIPS_SIM_ABI64
 #    define systemd_NR_renameat2 systemd_SC_arch_bias(311)
 #  endif
-#elif defined(__powerpc64__)
+#elif defined(__powerpc__)
```

Comment:
so, `__powerpc__` is always set when `__powerpc64__` is set, too, did I get this right?

Reply:
Right.